cloud-aws/src/main/resources/templates/aws-cf-stack.ftl

```diff
@@ -495,6 +495,25 @@
 
     <#if enableEfs>
     ,
+    "MountTargetSecurityGroup": {
+      "Type": "AWS::EC2::SecurityGroup",
+      "Properties": {
+        <#if existingVPC>
+          "VpcId" : { "Ref" : "VPCId" },
+        <#else>
+          "VpcId" : { "Ref" : "VPC" },
+        </#if>
+        "GroupDescription": "Security group for EFS mount target",
+        "SecurityGroupIngress": [
+          {
+            "IpProtocol": "tcp",
+            "FromPort": "2049",
+            "ToPort": "2049",
+            "CidrIp": "0.0.0.0/0"
```

Comment:
I suppose this SG is to control which clients may be able to connect to the mount target (https://docs.aws.amazon.com/efs/latest/ug/accessing-fs-create-security-groups.html). If so, why not restrict it to the CIDR of the subnet?

Reply:
This is AWS document regarding mount target:
A mount target provides an IP address for an NFSv4 endpoint at which you can mount an Amazon EFS file system. You mount your file system using its Domain Name Service (DNS) name, which resolves to the IP address of the EFS mount target in the same Availability Zone as your EC2 instance. You can create one mount target in each Availability Zone in an AWS Region. If there are multiple subnets in an Availability Zone in your VPC, you create a mount target in one of the subnets. Then all EC2 instances in that Availability Zone share that mount target. https://docs.aws.amazon.com/efs/latest/ug/how-it-works.html

I have talked with Haley, and she mentioned that we only support a single subnet within an availability zone. So I will make the change to use the existing subnet IP ranger to be more secure. Later on, if we support multiple subnet within an availability zone, we need to expend this range.

Reply:
I ended up using the VPC subnets as ingress. So it will allow access within the VPC. In this way, we don't need to change later, and still secure. For each sub-folder at the EFS, we have additional mechanism, such as access point or security group, to control the fine-grained access.